### PR TITLE
make `gulp size` write out json files instead of a txt file with tables

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: build-system/tasks/size.js [hunk 1]
```diff
@@ -14,35 +14,31 @@
  * limitations under the License.
  */
 
-var table = require('text-table');
 var del = require('del');
```
Comment (dvoytenko) [inline build-system/tasks/size.js]: Just one quick note - JSON was a suggestion. If you find it easier to do a text table - I'm all for it :)

**Comment by dvoytenko** [comment]
LGTM.

File: build-system/tasks/size.js (hunk 1 above)
Comment (erwinmombay) [inline build-system/tasks/size.js]: json is definitely easier to consume :D

File: test/sizes.gzip.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "amp.js": "92.33 kB",
```
Comment (cramforce) [inline test/sizes.gzip.json]: Its really sad to loose version history. Can't we read in the old format just as easily using a CSV reader?

Comment (dvoytenko) [inline test/sizes.gzip.json]: Good point. Version history here is nice... Should we rollback and try again?

Comment (erwinmombay) [inline test/sizes.gzip.json]: good point. we could keep it around too? i'm perfectly fine just reading the table too. (shouldnt be hard)

Comment (dvoytenko) [inline test/sizes.gzip.json]: What do you mean "keep it around"? I think the idea is to keep the sizes.txt as is and parse it as a pipe-delimited table. That way we get it to do what we want and keep the history so far as well.

Comment (erwinmombay) [inline test/sizes.gzip.json]: sorry, meant to keep the .txt file around and move forward with the .json.

will make the changes to keep the .txt.

Comment (dvoytenko) [inline test/sizes.gzip.json]: Yeah. Sorry for the switcheroo, but it does sound better at this point. We really don't need to have both.

Comment (erwinmombay) [inline test/sizes.gzip.json]: sgtm

**Pull request merged**